docker/webapp/Dockerfile

```diff
@@ -1,30 +1,38 @@
-FROM php:8.3-fpm-alpine
+FROM php:8.3-fpm-alpine AS basephp
 
-MAINTAINER Sarah Savage
-
-
-RUN export DEBIAN_FRONTEND=noninteractive \
-    && apk update \
-    && apk add \
-        bash \
-        linux-headers \
-        libpq-dev \
-        git \
-        zip \
-        libzip-dev \
-        icu-dev \
-        postgresql-client \
-        $PHPIZE_DEPS
+RUN apk add \
+    bash \
+    linux-headers \
+    libpq-dev \
+    git \
+    zip \
+    libzip-dev \
+    icu-dev \
+    postgresql-client \
+    $PHPIZE_DEPS
 
 
 RUN docker-php-ext-install pdo \
```

Comment:
This is good as-is, but I should point out mlocati/docker-php-extension-installer, which installs all the needed apt or apk dependencies then removes the build-time ones afterward.  See https://github.com/chazworks/waggypuppy/blob/main/docker/Dockerfile for an example of its use.

Reply:
Will take a deeper look but not going to hold ship on this.